augly/video/augmenters/ffmpeg/hflip.py

```diff
@@ -1,22 +1,35 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List
 
-from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from ffmpeg.nodes import FilterableStream
+from augly.video.augmenters.ffmpeg.base_augmenter import BaseVidgearFFMPEGAugmenter
 
 
-class VideoAugmenterByHFlip(BaseFFMPEGAugmenter):
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+class VideoAugmenterByHFlip(BaseVidgearFFMPEGAugmenter):
+    def get_command(self, video_path: str, output_path: str, **kwargs) -> List[str]:
         """
         Horizontally flips the video
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
+            in a command line
```

Comment:
could you remove this extraneous line? :D
```suggestion
            the augmentation
```